docs/user/tutorials/install_epinio_on_eks.md

```diff
@@ -0,0 +1,94 @@
+#  AWS EKS configuration
+
+## Create an EKS cluster
+
+If you don't have an existing cluster, follow the [quickstart](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html) to create an EKS cluster.
+
+## EKS Prerequisites
+
+* We follow the documented recommendation of having at least 2 nodes for EKS
+* Epinio v0.0.21 has been tested with AWS EKS incl. kubernetes version v1.20.4
+* To do more extensive testing we recommend a 2 node cluster with "t3.xlarge" instances
+* To just try out Epinio, e.g. 2 "t3a.large" are sufficient
+
+#### Install Dependencies
+
+Follow these [steps](./install_dependencies.md) to install dependencies.
+
+#### Install Epinio CLI
+
+* Download the binary
+
+Find the latest version from [Releases](https://github.com/epinio/epinio/releases) and run e.g.
+
+```bash
+curl -o epinio -L https://github.com/epinio/epinio/releases/download/v0.0.20/epinio-linux-amd64
```

Comment:
The thing you noticed. Version mismatch. Maybe use a placeholder like `VERSION` in the command, and note that this is the placeholder for the actual version.